src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Sarif.Multitool</id>
+    <version>$version$</version>
+    <title>Microsoft SARIF Multitool (includes SARIF SDK)</title>
+    <authors>Microsoft Corporation</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>SARIF-based classes and utilities for building static analysis drivers. Includes the SARIF SDK.</description>
```

Comment:
This comment is wrong (copy/pasted from the SDK).

Reply:
These were moved from the csproj. Do you have better values I can use?

Reply:
How about: `Multi-purpose tool for analyzing and manipulating SARIF files`

Reply:
Done